src/components/EventCell/styles.ts

```diff
@@ -1,22 +1,28 @@
-import { StyleSheet } from 'react-native';
-import { COPPER_GREY, WHITE, COPPER } from '../../styles/colors';
+import { StyleSheet, ViewStyle } from 'react-native';
+import { COPPER_GREY } from '../../styles/colors';
 import { FIRA_SANS_BOLD, FIRA_SANS_REGULAR } from '../../styles/fonts';
 import { BORDER_RADIUS, BORDER_WIDTH, MARGIN, PADDING } from '../../styles/metrics';
 
 const LEFT_BORDER_CELL = BORDER_WIDTH * 8;
 
-export default StyleSheet.create({
-  cell: {
-    display: 'flex',
-    flexDirection: 'row',
-    borderRadius: BORDER_RADIUS.MD,
-    borderWidth: BORDER_WIDTH,
-    borderLeftWidth: LEFT_BORDER_CELL,
-    borderColor: COPPER[100],
-    backgroundColor: WHITE,
-    marginHorizontal: MARGIN.MD,
-    paddingHorizontal: PADDING.LG,
-  },
+type eventCellType = {
+  borderColor: string,
+  backgroundColor: string,
+};
+
+const cellContainerStyle = ({ borderColor, backgroundColor } : eventCellType): ViewStyle => ({
```

Comment:
Pourquoi ne pas passer par `StyleSheet.create` ? Quitte à en avoir un deuxième ?

Reply:
Justement, c'est ce que ta methode propose de faire, ca revient a écrire: 
```
const styles = StyleSheet.create({
    cell: ({ borderColor, backgroundColor } : eventCellType): ViewStyle => ({
       display: 'flex',
       flexDirection: 'row',
       borderRadius: BORDER_RADIUS.MD,
       borderWidth: BORDER_WIDTH,
       borderLeftWidth: LEFT_BORDER_CELL,
       borderColor,
       backgroundColor,
       marginHorizontal: MARGIN.MD,
       paddingHorizontal: PADDING.LG,
  }),
});
```
Sauf qu'en écrivant ca, typescript renvoie une erreur car il faut que chaque element de style soit de type `ViewStyle`, `TextStyle` ou `ImageStyle`... C'est pour ca que je l'ai ecrit sous forme de méthode

Reply:
ça me va

Reply:
J'ai l'impression qu'en typant bien le retour de la methode on a équivalence avec l'utilisation de StyleSheet.create
https://stackoverflow.com/questions/38958888/react-native-what-is-the-benefit-of-using-stylesheet-vs-a-plain-object

Mais ca mériterait peut être de creuser un peu plus le sujet, je peux toujours revenir au style dynamique habituel en passant par `StyleSheet.create`, qu'en pensez-vous ?

Reply:
D'acc, je remet comme c'était au départ :)